src/components/nature-of-business/NatureOfBusiness.vue

```diff
@@ -180,24 +184,16 @@ export default class NatureOfBusiness extends Vue {
 
   /** Whether we have stored NAICS data. */
   get haveNaics (): boolean {
-    return (!!this.naicsCode && !!this.naicsDescription)
+    return (!!this.naicsCode || !!this.naicsDescription)
   }
 
   /** Whether this form is valid. */
   get isFormValid (): boolean {
     return (this.haveNaics && this.state === States.SUMMARY)
   }
 
-  /** Called when this component has been created. */
-  created (): void {
-    // on init, if we have stored NAICS data then display summary
-    if (this.haveNaics) {
-      this.state = States.SUMMARY
-    }
-  }
```

Comment:
The problem with this is the state is only updated when the component is created -- not when the props change.

Now we use a watcher instead (see below).